presto-base-jdbc/src/test/java/com/facebook/presto/plugin/jdbc/TestJdbcQueryBuilder.java

```diff
@@ -71,9 +75,12 @@ public void setup()
         PreparedStatement preparedStatement2 = connection.prepareStatement(stringBuilder.toString());
         preparedStatement2.execute();
 
-        cols.add(new JdbcColumnHandle("test_id", "col_0", BigintType.BIGINT));
-        cols.add(new JdbcColumnHandle("test_id", "col_1", DoubleType.DOUBLE));
-        cols.add(new JdbcColumnHandle("test_id", "col_2", BooleanType.BOOLEAN));
+        if (cols.isEmpty()) {
```

Comment:
Either make `cols` a constant, or create it here every time. Depending on state across `setup()` calls is confusing and brittle.